lib/ret_web/router.ex

```diff
@@ -120,6 +120,7 @@ defmodule RetWeb.Router do
       resources("/scenes", Api.V1.SceneController, only: [:create, :update])
       resources("/avatars", Api.V1.AvatarController, only: [:create, :update, :delete])
       resources("/hubs", Api.V1.HubController, only: [:update])
+      get("/rooms", Api.V1.HubController, :show)
```

Comment:
Let's just move Rooms to it's own controller. I expect we'll run both the HubController and RoomController for a while where RoomController is the public API and HubController will be deprecated.